docs/edit-profile.md

```diff
@@ -1,37 +1,48 @@
-# Edit profile
+# Edit your TechPass profile
 
-TechPass users can edit their TechPass profile anytime by logging into the TechPass portal using non-SE GSIB device.
+TechPass users can edit their TechPass profile anytime by logging into the TechPass portal.
 
-> **Note**
-> Create a [service request](https://go.gov.sg/techpass-sr) with us to update your profile details:
->- If you are a public officer with a SE GSIB device.
->- If you are a vendor or contractor whose TechPass ID's domain is *techpass.gov.sg*.
+?> Users who don't have access to TechPass portal, create a [service request](https://go.gov.sg/techpass-sr) with us to update your profile details.
 
-_To edit your profile:_
+## Audience
 
-1. From your non-SE GSIB device, go to [TechPass portal](https://portal.techpass.gov.sg) and click **Login with TechPass**.
-2. Sign in to your TechPass account.
-<kbd>![sign-in](assets/images/access-sgts-services-using-techpass/log-in-with-techpass.png)</kbd>
+TechPass users who can access TechPass portal.
 
-> **Note**
-> If you are a public officer you may be prompted to log in to the WOG account first before proceeding to TechPass account.
+## Prerequisites
 
-You will now be prompted to approve your TechPass sign-in. A number will be shown on your browser.
+You need the following to edit your TechPass profile via the TechPass portal:
 
- <kbd>![number-mfa](assets/images/onboarding/po-non-se/mfa-number-displayed-on-screen.png)</kbd>
+- An active TechPass account.
+- A non-SE GSIB device.
 
-3. On the Authenticator app, enter the number shown, and tap **Yes**.
+### To edit your TechPass profile
+
+1. From your non-SE GSIB device, log in to [TechPass portal](https://portal.techpass.gov.sg).
 
-4. Hover over your account name and click **My Account**.
+2. Hover over your account name and click **My Account**.
 
-<kbd>![view-account](assets/images/onboarding/po-non-se/view-account-or-profile.png)</kbd>
+![view-account](assets/images/onboarding/po-non-se/view-account-or-profile.png)
 
-5. Click **Edit Profile**.
+3. Click **Edit Profile**.
 
-<kbd>![edit-account](assets/images/onboarding/po-non-se/edit-profile-new.png)</kbd>
+![edit-account](assets/images/onboarding/po-non-se/edit-profile-new.png)
 
-6. Make the required changes and click **Save**.
+4. Make the required changes and click **Save**.
```

Comment:
The Agency/Company field cannot be changed anymore.

Reply:
Thanks, fixed the steps and screenshots as needed.